api/osrd_infra/schemas/infra.py

```diff
@@ -406,6 +374,9 @@ class BufferStop(BaseObjectTrait, TrackLocationTrait):
 
     applicable_directions: ApplicableDirections = Field(description="Direction of the application of the buffer stop")
 
+    def ref(self):
+        return DetectorReference(type="Detector", id=self.id)
```

Comment:
There must be something Ihave misunderstood, why is it not :
```suggestion
        return BufferStopReference(type="BufferStop", id=self.id)
```